### ISSUE TITLE
Please remove reference to "QMBuild"?

### ISSUE THREAD

**Issue by Jaykul** [questionmark/PowerShellTemplate#4]
The whole [Template.Tests](/master/tests/PowerShellTemplate.Tests.ps1) file seems like something Questionmark specific that accidentally slipped into the public branch ...

**Comment by JPRuskin** [comment]
I agree. I may replace that chunk of the file with a basic "tests with PSSA rules by default"?

**Comment by Jaykul** [comment]
In my personal projects, I've built a [ScriptAnalyzer-Job.yml](https://github.com/PoshCode/Azure-Pipelines/blob/master/ScriptAnalyzer-job.yml#L53-L96) but as we've discussed, we need to find a way to make these things runnable offline by devs as well, so it might be just as well to include them here ...

Maybe the best option is to throw out this file and replace it with three files, like:

PowerShellTemplate.ScriptAnalyzer.Tests.ps1
PowerShellTemplate.Help.Tests.ps1
PowerShellTemplate.Exports.Tests.ps1

And tag each of those with "Common", "Template" and with "ScriptAnalyzer","Help", and "Exports" respectively, so people can easily use or ignore them with tag support?

Ultimately I'd like to see some of these common tests adopted into Pester or a PoshCode Pester helper module 😉
